app/src/main/java/com/android/sample/ui/settings/SettingsScreen.kt

```diff
@@ -38,101 +41,100 @@ fun SettingsScreen(
               // User data button
-              Button(
+              BlueButton(
                   onClick = { navigationActions.navigateTo(Screen.EDIT_ACCOUNT) },
-                  modifier =
-                      Modifier.fillMaxWidth()
-                          .height(60.dp)
-                          .background(Color.LightGray, RoundedCornerShape(10.dp))
-                          .testTag("userDataButton"), // Test tag for User Data button
-                  colors = ButtonDefaults.buttonColors(containerColor = Color.LightGray)) {
-                    Text("User data", color = Color.Black)
-                  }
+                  modifier = Modifier.testTag("userDataButton"),
+                  title = R.string.UserData)
 
-              // Preferences button
-              Button(
+              BlueButton(
                   onClick = { navigationActions.navigateTo(Screen.PREFERENCES) },
-                  modifier =
-                      Modifier.fillMaxWidth()
-                          .height(60.dp)
-                          .background(Color.LightGray, RoundedCornerShape(10.dp))
-                          .testTag("preferencesButton"), // Test tag for Preferences button
-                  colors = ButtonDefaults.buttonColors(containerColor = Color.LightGray)) {
-                    Text("Preferences", color = Color.Black)
-                  }
+                  modifier = Modifier.testTag("preferencesButton"),
+                  title = R.string.Preferences)
+
+              Column(
+                  modifier = Modifier.fillMaxSize().padding(16.dp),
+                  verticalArrangement = Arrangement.Bottom,
+                  horizontalAlignment = Alignment.CenterHorizontally) {
+                    RedButton(
+                        onClick = {
+                          userAccountViewModel.deleteAccount(
+                              context,
+                              onSuccess = {
+                                Toast.makeText(
+                                        context,
+                                        R.string.SuccesfulDeleteMessage,
+                                        Toast.LENGTH_SHORT)
+                                    .show()
+                                navigationActions.navigateTo("Auth Screen")
+                              },
+                              onFailure = { error ->
+                                Toast.makeText(
+                                        context,
+                                        "Failed to delete account: ${error.message}",
+                                        Toast.LENGTH_LONG)
+                                    .show()
+                              })
+                        },
+                        image = Icons.Outlined.Delete,
+                        title = R.string.Delete_Account,
+                        modifier = Modifier.fillMaxWidth().testTag("deleteAccountButton"))
```

Comment:
Really good that you used R.String... I think I should also do that in my code !